cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/LanguageTraits.kt

```diff
@@ -230,3 +232,13 @@ interface HasFunctionalCasts : LanguageTrait
  * multiple functions can share the same name with different parameters.
  */
 interface HasFunctionOverloading : LanguageTrait
+
+/** A language trait that specifies that this language allows overloading of operators. */
+interface HasOperatorOverloading : LanguageTrait {
+
+    /**
+     * The name of the operators. The key is the [HasOperatorCode.operatorCode] and the value is the
+     * name of the operator.
+     */
+    val operatorNames: Map<String, Symbol>
```

Comment:
After initial confusion: Should we rename this to `overloadedOperatorNames`? @maximiliankaul

Reply:
Yes, please :)